### DISCUSSION TITLE
Imitating Mouse Wheel UP/DOWN

### DISCUSSION THREAD

**Discussion by Jonn3y** [rsjaffe/MIDI2LR#705] [Q&A]
Hello,

is there a way to imitate the mouse wheel up/down operations?

Greetings

Edit: shortcut code "cursor up" and "cursor down" seem to work for me. Sadly they do not responde as fast as my mouse. Or to say, they do not responde on every wheel click.

**Comment by paul-cine** [comment]
Keyboard maestro or bome
